### ISSUE TITLE
NVDA does not read actual Macro name when using Quick Access Toolbar or Customize Ribbon Functions

### ISSUE THREAD

**Issue by britechguy** [nvaccess/nvda#15315]
### Steps to reproduce:
1. Download this replacement [Normal.dotm](https://drive.google.com/uc?export=download&id=10Nm8KvKhUaqFwOyXGB3g5FsmswH-mdec) for testing purposes.  
2. 	Put that replacement file in the previously noted %APPDATA%\Microsoft\Templates folder for the person testing.
3.	Create a new blank document.  This will use the Normal.dotm you just put in as the replacement and that contains the macro.
4.	In MS-Word, open the Customize Quick Access option using ALT, Up Arrow.  If you hear either the Document name or place holder, then use left arrow, if you hear Autosave On, then then right arrow until you hear, “Customize Quick Access Toolbar, Collapsed.” Expand it.
5.	Hit Up Arrow 3 times to navigate to the More Commands menu item, and activate it.
6.	Hit TAB once, and you’ll be on the Choose commands from dropdown.  Expand it and choose Macros from the list.
7.	Hit Tab and you will be in the Macro list, and on an item for Separator.  Hit down arrow once and you will be on the macro for using the traditional spell checker, which should be “List Item 2 of 2” since the separator is item 1. NOTE:  NVDA does not announce any name for this.  In Issue #7087 from 2017, user @Adriani90 states that JAWS does announce the name of the macro.
8.	Navigate to the Add button, and activate it.  This will cause a button for triggering that macro to be added to your Quick Access Toolbar.  If you have not added, or removed, anything from the Quick Access Toolbar in the past, it should have the shortcut ALT + 5.  Oddly enough, the button will announce the fully qualified Macro name, which ends with UseTraditionalSpellCheck dialog.

NOTE:  NVDA does announce the name of the macro when focus is on the button created in the Quick Access Toolbar.

### Actual behavior:
Macro name is not announced, only "Item 2 of 2."

### Expected behavior:
Macro name itself should be announced when traversing the list

### NVDA logs, crash dumps and other attachments:
N/A

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2023.1
#### Windows version:
Windows 11 Home Version 22H2
#### Name and version of other software in use when reproducing the issue:
Word 365
#### Other information about your system:
LG Gram 16 with i5 12th Gen and 16 GB RAM
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Have run, but do not have time right now to restart.

**Comment by Adriani90** [comment]
Testing with NVDA alpha-53423,57f2e9dc (2026.1.0.53423)
and Microsoft 365 MSO (Version 2510 Build 16.0.19328.20190) 64 Bit

This issue does not occur anymore. Macro names are reported properly in this list.
Closing as works for me.
